src/components/ToolBar/DataMenu/SaveWorkspaceToNDEx.tsx

```diff
@@ -128,17 +128,16 @@ export const SaveWorkspaceToNDExMenuItem = (
       )
 
       const workspace = await getWorkspaceFromDb(currentWorkspaceId)
+      const onlyNdexNetworkIds = workspace.networkIds.filter(id => summaries[id]?.isNdex !== false);
```

Comment:
I think it is better to change `id => summaries[id]?.isNdex !== false` to `id => summaries[id]?.isNdex ===true`.

The reason for that is: 

- `summaries[id]?.isNdex === true `explicitly checks that the value of isNdex is true (i.e., the boolean true).
- `summaries[id]?.isNdex !== false `is broader and includes any value that is not strictly `false`, such as `undefined`, `null`